src/test/java/guitests/UseSessionConfigsTest.java

```diff
@@ -163,8 +164,16 @@ public void useGlobalConfigTest() {
         assertEquals("Dummy 2 panel", dummyBoard.get(2).getPanelName());
 
         // Panels
-        List<String> lastOpenFilters = testPref.getLastOpenFilters();
-        List<String> lastOpenPanelNames = testPref.getPanelNames();
+        List<String> lastOpenFilters =
+                testPref.getPanelInfo()
+                        .stream()
+                        .map(PanelInfo::getPanelFilter)
+                        .collect(Collectors.toList());
+        List<String> lastOpenPanelNames =
+                testPref.getPanelInfo()
+                        .stream()
+                        .map(PanelInfo::getPanelName)
+                        .collect(Collectors.toList());
```

Comment:
Just wondering. The two methods on the left is still in `SessionConfig`, although their facade in `Preferences` was removed. Any reason for that? If we don't use the methods, we need to remove the ones in `SessionConfig` as well.